server/index.js

```diff
@@ -34,6 +35,9 @@ const {
 
 const { sessionKey } = require('@private/authConfig.json');
 
+// configure.js doesn't handle new keys in config.json files yet. Make sure it doens't break.
+let bLE;
```

Comment:
I suggest using more descriptive variable names.
Keep names succinct and a descriptive label, even `currentBlocklistEndpoint` would work